bcc/module.go

```diff
@@ -114,9 +114,11 @@ func (bpf *Module) Close() {
 	C.bpf_module_destroy(bpf.p)
 	for k, v := range bpf.kprobes {
 		C.perf_reader_free(v)
-		desc := fmt.Sprintf("-:kprobes/%s", k)
-		descCS := C.CString(desc)
-		C.bpf_detach_kprobe(descCS)
+		descCS := C.CString(k)
```

Comment:
In `attachProbe` we we have named it `evNameCS` (in line with `ev_name` from libbpf.h). Could you rename to `evNameCS` here as well?